examples/fonts-custom/docs/README.md

```diff
@@ -0,0 +1,196 @@
+      src: url("https://cdn.jsdelivr.net/gh/notofonts/notofonts.github.io/fonts/NotoSerif/unhinted/otf/NotoSerif-Regular.otf");
+    }
+
+    @font-face {
+      font-family: "Noto Serif"; font-weight: bold; font-style: normal;
+      src: url("https://cdn.jsdelivr.net/gh/notofonts/notofonts.github.io/fonts/NotoSerif/unhinted/otf/NotoSerif-Bold.otf");
+
+    }
+
+    @font-face {
+      font-family: "Noto Serif"; font-weight: normal; font-style: italic;
+      src: url("https://cdn.jsdelivr.net/gh/notofonts/notofonts.github.io/fonts/NotoSerif/unhinted/otf/NotoSerif-Italic.otf");
+
+    }
+
+    @font-face {
+      font-family: "Noto Mono"; font-weight: normal; font-style: normal;
+      src: url("https://cdn.jsdelivr.net/gh/notofonts/notofonts.github.io/fonts/NotoSansMono/unhinted/otf/NotoSansMono-Regular.otf");
+
+    }
+
+    @font-face {
+      font-family: "Noto Mono"; font-weight: bold; font-style: normal;
+      src: url("https://cdn.jsdelivr.net/gh/notofonts/notofonts.github.io/fonts/NotoSansMono/unhinted/otf/NotoSansMono-Bold.otf");
+    }
+
+    :root {
+      --md-text-font: "Noto Serif";
+      --md-code-font: "Noto Mono";
+    }
+    ```
+
+### Hosting on your own site
+
+The code for hosting the fonts yourself as part of your site, looks
+very similar to the previous example that used a CDN. You need to
+write your own font face definitions but this time you change the URL
+to point to the files located on your own server, e.g., in
+`docs/assets/fonts`:
+
+=== "`mkdocs.yml`"
+
+    ```yaml
+    extra_css:
+      - assets/stylesheets/extra.css
+    ```
+
+=== "`extra.css`"
+
+    ```css
+    @font-face {
+      font-family: "Noto Serif"; font-weight: normal; font-style: normal;
+      src: url("https://cdn.jsdelivr.net/gh/notofonts/notofonts.github.io/fonts/NotoSerif/unhinted/otf/NotoSerif-Regular.otf");
+    }
+
+    @font-face {
+      font-family: "Noto Serif"; font-weight: bold; font-style: normal;
+      src: url("https://cdn.jsdelivr.net/gh/notofonts/notofonts.github.io/fonts/NotoSerif/unhinted/otf/NotoSerif-Bold.otf");
+
+    }
+
+    @font-face {
+      font-family: "Noto Serif"; font-weight: normal; font-style: italic;
+      src: url("https://cdn.jsdelivr.net/gh/notofonts/notofonts.github.io/fonts/NotoSerif/unhinted/otf/NotoSerif-Italic.otf");
+
+    }
+
+    @font-face {
+      font-family: "Noto Mono"; font-weight: normal; font-style: normal;
+      src: url("https://cdn.jsdelivr.net/gh/notofonts/notofonts.github.io/fonts/NotoSansMono/unhinted/otf/NotoSansMono-Regular.otf");
+
+    }
+
+    @font-face {
+      font-family: "Noto Mono"; font-weight: bold; font-style: normal;
+      src: url("https://cdn.jsdelivr.net/gh/notofonts/notofonts.github.io/fonts/NotoSansMono/unhinted/otf/NotoSansMono-Bold.otf");
+    }
+
+    :root {
+      --md-text-font: "Noto Serif";
+      --md-code-font: "Noto Mono";
+    }
+    ```
+
+
+## (Dis-)Advantages
+
+If you are loading the fonts from a CDN as in this example, you are
```

Comment:
One final question: what line gutters are you using? I'm trying to limit everything to 80 chars, as it allows to render code blocks in PDFs nicely, but it looks like you're using 70 or something in that area. Could we agree on 80? Sorry to be so pendantic 😅

Reply:
Indeed, it seems that for some reason it is 70 on `.md` files. This is `vim`'s doing. I have changed it to 80. Shame this does not seem to be something that a `.editorconfig` could take care of?!

Reply:
I am also just realizing that I do not understand the comment about code blocks. Those sometimes get longer, especially when URLs are involved. I am not sure what to do about that.

You can stop apologizing for being "pedantic" - that is quite ok and we know each other well enough by now!

Reply:
> Indeed, it seems that for some reason it is 70 on .md files. This is vim's doing. I have changed it to 80. Shame this does not seem to be something that a .editorconfig could take care of?! Do you want me to change these or just use 80 from here onwards?

Please just use 80 onwards. When we revisit those docs at some point, we can correct them the next time.

> I am also just realizing that I do not understand the comment about code blocks. Those sometimes get longer, especially when URLs are involved. I am not sure what to do about that.

When you print a site made with Material for MkDocs from the browser, limiting code blocks to 80 characters will always nicely fit the page. That's one of the reasons I always stick to the 80 characters rule. Additionally, it allows to open multiple editor tabs next to each other without having to scroll horizontally. I'm just not that big a fan of horizontal scrolling on text ☺️ So to sum up: 80 is just the best practice I've been following for many years.

> You can stop apologizing for being "pedantic" - that is quite ok and we know each other well enough by now!

Ack!

Reply:
I don't question the 80 chars rule (I had 70 configured, probably on the basis of recommended line length for reading), just wondering what to do with things like this in the CSS file:

```
https://cdn.jsdelivr.net/gh/notofonts/notofonts.github.io/fonts/NotoSerif/unhinted/otf/NotoSerif-Bold.otf
```